### DISCUSSION TITLE
Driver not found

### DISCUSSION THREAD

**Discussion by lublum** [dunglas/symfony-docker#198] [General]
Hi,

I had a lot of issues with a missing postgresql driver, for example "Driver not found" when I tried to execute make:migrations. Finally resolved this by changing line 183 of the **Dockerfile**

from
`docker-php-ext-install -j$(nproc) pdo_pgsql; \`

to
`docker-php-ext-install -j$(nproc) pdo pgsql pdo_pgsql; \`

What do you think?

Cheers
Ludmila

**Comment by maxhelias** [comment]
Hi!
We do not provide "direct support" for postgres in this template. We extend it with Flex see : https://github.com/dunglas/symfony-docker/blob/main/docs/extra-services.md and https://github.com/symfony/recipes/blob/master/doctrine/doctrine-bundle/2.4/manifest.json#L17-L21

**Reply by lublum** [reply]
Ah, I see! Thank you!
